client/src/productOverview/overviewApp.jsx

```diff
@@ -36,15 +37,19 @@ const OverviewApp = (props) => {
       {(overviewProduct === null || allStyles === null || currentStyle === null)
         ? <div>Loading</div>
         :
-        <div className="overview-container">
+        <div className="overview-container" id="overview-container">
```

Comment:
You may want to decide on either classname or using id here since it's duplicative.
If there is only one overview-container, then and id would probably be the right choice.
Alternatively you can keep className and when you need to select it, getElementsByClassName('overview-container')[0].